internal/cluster/cluster.go

```diff
@@ -982,9 +982,11 @@ func (m Manager) PermanentClustersDeletion(ctx context.Context, olderThan strfmt
 		return reply.Error
 	}
 	for i := range clusters {
+
 		c := clusters[i]
-		m.log.Debugf("Deleting all S3 files for cluster: %s", c.ID.String())
+		m.log.Infof("Permanently deleting cluster %s that was de-registered before %s", c.ID.String(), olderThan)
 
+		m.log.Debugf("Deleting all S3 files for cluster: %s", c.ID.String())
```

Comment:
why do we need the debug log when we have the above info log?

Reply:
I guess it's redundant to be specific about what is being deleted,